client/router_create.go

```diff
@@ -621,21 +621,21 @@ func (cli *VanClient) GetRouterSpecFromOpts(options types.SiteConfigSpec, siteId
 			},
 		},
 		})
-		envVars = append(envVars, corev1.EnvVar{Name: "QDROUTERD_AUTO_MESH_DISCOVERY", Value: "QUERY"})
+		envVars = append(envVars, corev1.EnvVar{Name: "SKROUTERD_AUTO_MESH_DISCOVERY", Value: "QUERY"})
```

Comment:
Same for all these environment variables. They must be changed in `skupper-router` first.